src/main/java/com/gmail/model/User.java

```diff
@@ -0,0 +1,34 @@
+package com.gmail.model;
+
+import com.gmail.annotations.InjectRandomInt;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.time.LocalDate;
+
+/**
+ * Created by Space on 09.05.2019.
+ */
+@Data
+@Setter
+@Getter
+@Entity
+@Component
```

Comment:
Why it is a component? It is just an entity. Please fix.

Reply:
the user must be a component otherwise the annotation @InjectRandomInt will not work